doc/install_ubuntu2004_VM.sh

```diff
@@ -155,6 +155,11 @@ mkdir -p Codes
 	cd $Codes/PHiLiP ;\
 	git checkout master ;\
 	git remote add upstream https://github.com/dougshidong/PHiLiP.git ;\
+    git config submodule.recurse true ;\
+    
+    # Get mesh files of NACA 0012. Note: Run get_NACA0012_mesh_files_cluster.sh to get files on the cluster.
+    sh get_NACA0012_mesh_files_local.sh ;\
```

Comment:
Be careful here, need to add lines for installing `gdown` before calling `get_NACA0012_mesh_files_local.sh`; see `INSTALL.md`